suite/tests/CMakeLists.txt

```diff
@@ -2819,31 +2833,43 @@ if (NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
   use_DynamoRIO_extension(client.tls.dll drmgr)
 endif (NOT RISCV64)
 
-if (X86)
+if (X86 OR AARCH64)
   if (NOT MACOS)
     # XXX i#2985: The test's asm doesn't build with MacOS's clang.
-    tobuild_ci(client.drx-scattergather client-interface/drx-scattergather.c "" "" "")
+    set(client.drx-scattergather_client_source "client-interface/drx-scattergather.dll.c")
```

Comment:
Do we expect to be able to reuse a lot of code from this dll file?

Reply:
I'm not sure I understand what you are asking. The `drx-scattergather.dll` test client is used for both the x86 and AArch64 versions of the test. There are a few x86-specific bits which are `#ifdef`ed out but the rest is shared.

Reply:
I was trying to weigh whether it will be cleaner to create a new dll test client file for aarch64, but let's keep the ifdefs and the same file for now.